content/en/docs/concepts/_index.md

```diff
@@ -19,8 +19,8 @@ To work with Kubernetes, you use *Kubernetes API objects* to describe your clust
 
 Once you've set your desired state, the *Kubernetes Control Plane* works to make the cluster's current state match the desired state. To do so, Kubernetes performs a variety of tasks automatically--such as starting or restarting containers, scaling the number of replicas of a given application, and more. The Kubernetes Control Plane consists of a collection of processes running on your cluster: 
 
-* The **Kubernetes Master** is a collection of three processes that run on a single node in your cluster, which is designated as the master node. Those processes are: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) and [kube-scheduler](/docs/admin/kube-scheduler/).
-* Each individual non-master node in your cluster runs two processes:
+* The **Kubernetes Master** is a collection of three processes that run on a single machine in your cluster, which is designated as the master. Those processes are: [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager](/docs/admin/kube-controller-manager/) and [kube-scheduler](/docs/admin/kube-scheduler/).
```

Comment:
if we are doing such a big refactor we might as well change all instances of `master` to `control plane` as this is a Steering Committee recommendation.
it was decided that `master` is a word we should avoid.

Reply:
> ... this is a Steering Committee recommendation...

You don't happen to have a link to that descision do you?  Would be nice to drop it into the comit message.  And is the recommended wording "control-plane machine"?  To me, "control plane" sounds like something that a cluster has one of, and includes load balancers, etc. involved in serving the Kubernetes API and extention APIs.

Reply:
it's in a VOD somewhere. i was informed about it in one of the SIG Cluster Lifecycle meetings.

> And is the recommended wording "control-plane machine"? 

it's a "control plane node", a machine can have multiple nodes depending on the solution at play.

> To me, "control plane" sounds like something that a cluster has one of

a cluster can have many control plane nodes.

Reply:
> it's a "control plane node"...

But nodes can schedule pods.  So control-plane node would be fine for, say, a self-hosted control plane where AWS instances (or whatever) have `kubelet`s with a `master` role.  But it wouldn't apply to instances that ran control-plane components natively (without a container runtime) and did not run a `kubelet` or otherwise provide for scheduling pods.

> ... a machine can have multiple nodes depending on the solution at play.

That's fine, isn't it?  Should we talk about clusters like "three control-plane machines and four compute machines" if they have seven AWS instances (or equivalent).  Such clusters could have anywhere from four to seven nodes (depending on whether control-plane machines ran connected `kubelets`) and could even have more nodes if you wanted to run multiple `kubelet`s per-machine or similar.

Reply:
> But it wouldn't apply to instances that ran control-plane components natively (without a container runtime) and did not run a kubelet or otherwise provide for scheduling pods

running a control plane component "natively" would still qualify said node as a control plane node.

>  Such clusters could have anywhere from four to seven nodes (depending on whether control-plane machines ran connected kubelets) and could even have more nodes if you wanted to run multiple kubelets per-machine or similar.

Reply:
> running a control plane component "natively" would still qualify said node as a control plane node.

How would it qualify as a node if you can't scedule pods on it?

> arguably a node is not a cluster element where a kubelet has to runs, even tough that's how the official docs outline it.

Doesn't have to be `kubelet`, but does have to be schedulable.  Or did you have another definition in mind?

Reply:
regarding the link to decision, please see
- https://github.com/kubernetes/website/issues/6525
- "February 1, 2018" https://github.com/kubernetes/community/blob/master/communication/meeting-notes-archive/q1-2_2018_community_meeting_minutes.md

Reply:
> How would it qualify as a node if you can't scedule pods on it?

Reply:
> it can still be a point of connectivity in the cluster.

This is a nuch broader definition than the current glossary entry, and it diverges (I think?) from `kubectl get nodes`.  But did you want me to reroll this PR to take this broader approach to "node"?

Reply:
> This is a much broader definition than the current glossary entry, and it diverges (I think?) from kubectl get nodes

definitely, there are topologies that the current definition does not cover.

> But did you want me to reroll this PR to take this broader approach to "node"?

no, i'm mostly insisting that we drop the usage of "master" and use "control plane (node)" instead.

Reply:
> definitely, there are topologies that the current definition does not cover.

This seems more like "I'm overloading *nodes*" ;).  Wouldn't `kubectl get nodes` (and [`NodeInterface`][1], [`NodeStatus`][2], etc.) be the root of our node definition?  I'd much not include unschedulable things under "nodes" and just use "points of connectivity" or similar if you really need to refer to such a generic concept.

But I'll reroll this PR within the hour to make the changes that we all agree on so far ;).

[1]: https://godoc.org/k8s.io/client-go/kubernetes/typed/core/v1#NodeInterface
[2]: https://godoc.org/k8s.io/api/core/v1#NodeStatus

Reply:
thank you.